static/app/views/settings/organizationDeveloperSettings/sentryApplicationRow/sentryApplicationRowButtons.tsx

```diff
@@ -40,6 +68,22 @@ function SentryApplicationRowButtons({
           );
         }
 
+        // feature flag new publishing flow features
+        if (organization.features.includes(`streamlined-publishing-flow`)) {
+          if (hasInvalidStatus(app)) {
+            disablePublishReason = t('Only unpublished integrations can be published');
```

Comment:
I think we should provide a more specific message for when the integration is in the review stage, since this message suggests the integration has been successfully published already